panel/io/reload.py

```diff
@@ -101,16 +105,20 @@ async def async_file_watcher(stop_event=None):
             continue
         if path.endswith((".pyc", ".pyo")):
             path = path[:-1]
-        modules[path] = module_name
+        path = str(pathlib.Path(path).resolve())
```

Comment:
```suggestion
        path = os.path.abspath(path) 
```

Alternative: `os.path.abspath(os.path.realpath(path))`